install/Knative-with-Minishift.md

```diff
@@ -10,37 +10,64 @@ You can find [guides for other platforms here](README.md).
 
 ## Before you begin
 
-These instructions will run an OpenShift 3.10 (Kubernetes 1.10) cluster on your
+These instructions will run an OpenShift 3.11 (Kubernetes 1.11) cluster on your
 local machine using [minishift](https://docs.okd.io/latest/minishift/getting-started/index.html)
 to test-drive Knative.
 
+## Minishift setup
+
```

Comment:
Maybe also an information about setup of virtualization providers would be useful here: https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html. Something like:

- [Prepare virtualization environment](https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html) to be used with Minishift

Reply:
+1

Reply:
IMO we do not need the `## Before you begin` section here. It should be just `## Minishift setup` and we can point to  https://docs.okd.io/latest/minishift/getting-started/index.html and ask users to follow all the steps mentioned in that page.  Putting non idex.html is not preferable as the web address will change if the name of the page changes (which we have done in the past). So just pointing to the index.html is safer even if we change the layout of the docs.